values-schema.yaml

```diff
@@ -207,6 +221,68 @@ definitions:
       tag:
         type: string
     title: Container image
+    type: object
+  job:
+    additionalProperties: false
+    properties:
+      enabled:
+        default: true
+        type: boolean
+      env:
+        $ref: '#/definitions/env'
+      image:
+        $ref: '#/definitions/image'
+      init:
+        properties:
+          image:
+            $ref: '#/definitions/image'
+          script:
+            $ref: '#/definitions/script'
+          shell:
+            pattern: (sh|bash)
+      mounts:
+        $ref: '#/definitions/mounts'
+      name:
+        default: job-hello-world
+        description: |
+          Must be a string starting with 'job-'. Name must be globally unique.
+        pattern: ^job-.+$
+        type: string
+      resources:
+        $ref: '#/definitions/resources'
+      script:
+        $ref: '#/definitions/script'
+      secret:
+        $ref: '#/definitions/secret'
+      serviceAccountName:
+        description: |
+          May be a non-empty string containing the name of the serviceAccount
+          that is authorized to launch the containers that will execute the job. 
+          Defaults to 'default' Service Account.
+        type: string
+      ttlSecondsAfterFinished:
+        pattern: ^[0-9]*$
+        type: number
+      type:
+        default: job
+        pattern: (job|cronjob)
+        type: string
+      volumes:
+        $ref: '#/definitions/volumes'
+    required:
+      - enabled
+      - image
+      - name
+      - script
+      - type
+
+  jobs:
+    description: You may pass an array of Job objects. The Kubernetes API accepts
+      a Job specification, which is meant for run-to-completion apps (e.g. batch
+      Jobs). Job names must be globally unique.
```

Comment:
globally? really? I think it needs to be unique in the namespace

Reply:
Sounds good, I now think that sentence doesn't makes sense here...